### PR TITLE
Update create-config-file.md

### PR DESCRIPTION
**Pull request by marybelvargas**
I believe the WAL is NOT required but encouraged. If it is required then might be worth to update the wording on this statement. Probably something like:  

> This is required and it ensures that samples will be redelivered in case of failure (e.g., network issues, machine reboot).

### CODE REVIEW

**Review by clayton-cornell — APPROVED** [review]
Agreed. This should state "not required" just to make the sentence and concept make sense.

**Comment by clayton-cornell** [comment]
@grafana/grafana-agent-core-maintainers Can someone on the team verify that WAL is NOT required (but is useful) with Agent on Windows?

**Comment by mattdurham** [comment]
A WAL is essentially required in the Agent. Technically if you are NOT doing metrics then you don't need a WAL, but given that metrics are our most common use case I would say it is required in Agent Static Mode.

**Comment by mattdurham** [comment]
In rereading this there are some statements that are incorrect with the original text. A WAL currently does not protect against machine reboots/agent restarts. Its an open issue with Prometheus.

File: docs/sources/static/configuration/create-config-file.md
Comment (clayton-cornell) [inline docs/sources/static/configuration/create-config-file.md:45]:
```suggestion
store metrics in a Write-Ahead Log (WAL). WAL is required and ensures that samples
```

File: docs/sources/static/configuration/create-config-file.md
Comment (clayton-cornell) [inline docs/sources/static/configuration/create-config-file.md:46]:
```suggestion
will be redelivered in case of failures, for example if there are network connectivity issues. We
```

@mattdurham would this suggestion cover the point you raised about reboots/restarts not handled by WAL?

Comment (mattdurham) [reply docs/sources/static/configuration/create-config-file.md:46]: Yes, that is primary use case.

**Comment by clayton-cornell** [comment]
@marybelvargas Over to you with the suggestions (combined with Matt's input). If you're happy with the suggestions (and apply them), ping me back and I can finalize and merge.

**Comment by marybelvargas** [comment]
Hi @clayton-cornell,

I'm good with the changes. Thank you!!

**Comment by clayton-cornell** [comment]
Merging

**Comment by clayton-cornell** [comment]
I am closing this PR because the CLA has not been signed by everyone. I cannot merge this PR without the CLA requirement. 

I've included this change in another PR: https://github.com/grafana/agent/pull/4355 that I'm working on so the correction is not lost/forgotten.

**Pull request closed without merging**
